deployment/docker-compose-production.yml

```diff
@@ -34,5 +34,12 @@ services:
       - auth-production:/var/lib/postgresql/data
     env_file:
       - ../stack.env
+  backup:
+    image: offen/docker-volume-backup:v2.27.0
+    restart: always
+    env_file:
+      - ../backup.env
```

Comment:
Could we add the stack env file here as well so we can backup secrets that are not commited

Reply:
Oh I should have added it in the first place because the path/aws credentials would be read from there :facepalm: But adding stack.env here wouldn't cause the secrets in stack.env to be backed up. I think to do that you'd need to dump the env into a file and get that into the /backup, but it's not configurable (I don't think it's the intended use case)

Reply:
I see, I think that a trade-off we probably need to take since that is how Portainer uses env for different stacks.